packages/agoric-cli/src/commands/oracle.js

```diff
@@ -43,14 +41,20 @@ export const makeOracleCommand = async logger => {
   `,
   );
 
-  const lookupPriceAggregatorInstance = ([brandIn, brandOut]) => {
-    const name = `${brandIn}-${brandOut} price feed`;
-    const instance = agoricNames.instance[name];
-    if (!instance) {
-      logger.debug('known instances:', agoricNames.instance);
-      throw new Error(`Unknown instance ${name}`);
-    }
-    return instance;
+  const rpcTools = async () => {
+    const { agoricNames, fromBoard, vstorage } = await makeRpcUtils({ fetch });
```

Comment:
consider letting the object pass through,
```suggestion
    const utils = await makeRpcUtils({ fetch });
```
and then,
```js
const instance = utils.agoricNames.instance[name];
…
return {...utils, lookupPriceAggregatorInstance};
```